### ISSUE TITLE
Definition of loss function

### ISSUE THREAD

**Issue by JianWenJun** [lancopku/SGM#1]
Hello, can you explain that what is the meaning of the 'sim_score' parameter in the 'cross_entropy_loss' function of the 'models/loss.py' file.  I am confused about this. thanks.
